src/AppBundle/Form/MembershipChooseNearbyCommitteeType.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace AppBundle\Form;
+
+use AppBundle\Entity\Committee;
+use Doctrine\ORM\EntityRepository;
+use Symfony\Bridge\Doctrine\Form\Type\EntityType;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\SubmitType;
+use Symfony\Component\Form\FormBuilderInterface;
+
+class MembershipChooseNearbyCommitteeType extends AbstractType
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('committees', EntityType::class, [
+                'class' => Committee::class,
+                'query_builder' => function (EntityRepository $er) {
+                    return $er
+                        ->createQueryBuilder('c')
+                        ->setMaxResults(3)
+                    ;
+                },
+                'choice_label' => 'name',
+                'multiple' => true,
+                'expanded' => true,
+            ])
+            ->add('submit', SubmitType::class, [
```

Comment:
Please add this button in the controller instead, or use an option to add conditionally, to ease reusability of the type.